implementations/rust/ockam/ockam_command/src/tcp/listener/show.rs

```diff
@@ -0,0 +1,57 @@
+use clap::Args;
+
+use crate::node::default_node_name;
+use crate::util::extract_address_value;
+use ockam::Context;
+use ockam_api::nodes::models;
+use ockam_core::api::Request;
+
+use crate::util::{node_rpc, Rpc};
+use crate::CommandGlobalOpts;
+
+#[derive(Clone, Debug, Args)]
+pub struct ShowCommand {
+    #[command(flatten)]
+    node_opts: TCPlistenerNodeOpts,
+
+    /// Tcp Listener ID
+    pub id: String,
+}
+
+#[derive(Clone, Debug, Args)]
+pub struct TCPlistenerNodeOpts {
+    /// Node at which to create the listener
+    #[arg(global = true, long, value_name = "NODE", default_value_t = default_node_name())]
+    pub at: String,
+}
```

Comment:
You can simplify this and inline the `at` argument within the `ShowCommand` struct.